delfin/tests/unit/drivers/test_api.py

```diff
@@ -253,63 +253,39 @@ def test_list_volumes(self, driver_manager, mock_fake):
         mock_fake.assert_called_once()
 
     @mock.patch.object(FakeStorageDriver, 'list_controllers')
-    @mock.patch('delfin.db.storage_create')
```

Comment:
As discussed please update the reason for changes required in this UTs when updated with 0.8.0 changes..

Reply:
Found the reason from commit id 4d8429da1ccbc95d677c39d7b0f708d1001f1853 . driver/manager.py has change on the logic of selecting driver. this casued update in all driver/api testcases. same change had to be done for new driver resource interfaces when this commit is pulled to mater